### PR TITLE
Feature/recipe instructions ingredients

### PR DESCRIPTION
**Pull request by tfettig22**
What’s this PR do?
- Add functionality to separate the instructions list into a numbered list, populate the ingredients section and the price section.  I also added a method to the Recipe class so that the cost of each ingredient is displayed in dollars.  And noticed that the o was not capitalized in one of our methods and so updated that accordingly and made sure all tests are still passing.
Where should the reviewer start?
- Start by double checking the viewRecipe function. It looks big and ugly for now but we will separate it out into smaller helper functions eventually. I did some slight refactoring of the Recipe methods, but ensured all tests still pass.
How should this be manually tested?
- Pull branch down and run npm test to verify, and npm start and view the live page to make sure everything looks correct.
Any background context you want to provide?
- Still need to separate the viewRecipe function into smaller chunks but it all works as intended so far.  We also should think about re-organizing the ingredients/price box so that each ingredient has the price clearly matched with it.

### CODE REVIEW

File: dist/index.html [hunk 1]
```diff
@@ -17,49 +26,29 @@ <h1>ANYONE CAN COOK</h1>
...
-          <img class="icon-4-img" src="">
-        </div>
-      </section>
-    </section>
-  </section>
-    <section class ="recipe-container hidden" id="recipeContainer">
-      <article class ="recipe-r-side-box" id="recipeRSideBox">
-        <p class ="recipe-title-box" id="recipeTitleBox">innerHTML recipe title</p>
-        <div class ="recipe-box" id="recipeBox">recipeInfo</div>
-      </article>
```
Comment (GOECHA) [inline dist/index.html:57]: Removed duplicates - reviewed

File: dist/index.html [hunk 2]
```diff
@@ -17,49 +26,29 @@ <h1>ANYONE CAN COOK</h1>
...
+            </form>
+          </div>
+          <div class="radio-wrapper">
+            <input class = "tag-search" type = "radio" name = "searchMode" value = "tag">Search by Tag</input>
+            <input class = "name-search" type = "radio" name = "searchMode" value = "name">Search by Name</input>
+          </div>
+        </article>
       <section class ="main-page-container hidden" id="mainPageContainer">
         <div class ="feature-container" id="featureContainer">
-          <div class="feature-image-box" id="featureImageBox"></div>
```
Comment (GOECHA) [inline dist/index.html:60]: Removed buzz container for styling

File: src/scripts.js [hunk 3]
```diff
@@ -70,8 +77,15 @@ function updateMainPageRecipeIcons() {
   icon2Img.src = allRecipes[getRandomIndex(allRecipes)].image;
   icon3Img.src = allRecipes[getRandomIndex(allRecipes)].image;
   icon4Img.src = allRecipes[getRandomIndex(allRecipes)].image;
+  icon5Img.src = allRecipes[getRandomIndex(allRecipes)].image;
+  icon6Img.src = allRecipes[getRandomIndex(allRecipes)].image;
 }
 
+function updateMainPaigeFeatureImg(){
```
Comment (forsethnico) [inline src/scripts.js:84]: There is a small typo in the spelling of this function. "page" has an "i" in it

**Review by forsethnico — APPROVED** [review]
Added comment for small spelling typo in updateMainPaigeFeatureImg() function on scripts.js. Like the adjusted order of the HTML and the CSS is looking great!

File: src/scripts.js [hunk 4]
```diff
@@ -81,38 +95,55 @@ function displayRecipeNames(recipeData) {
...
+  });
+// put code above into separate helper function
+  ingredientBox.innerHTML = '';
+  selectedRecipe[0].getIngredientNames().forEach(ingredient => {
+    ingredientBox.innerHTML += `<p class='recipe-ingredients'> ${ingredient} </p></br>`
+  });
+// put code above into separate helper function
+  priceListBox.innerHTML = '';
+  selectedRecipe[0].getCostOfIngredientsInDollars().forEach(cost => {
+  priceListBox.innerHTML += `<p class='ingredient-prices'> ${cost} </p></br>`
```
Comment (GOECHA) [inline src/scripts.js:127]: Formatting for innerHTML- reviewed- great job! Looks fantastic on user story 2 selected recipe.

**Review by GOECHA — APPROVED** [review]
Great job formatting innerHTML and helping assist with header bar movement through user stories. Code looks good.

File: src/scripts.js (hunk 3 above)
Comment (tfettig22) [reply src/scripts.js]: typo fixed, thanks for catching this!

**Pull request merged**